cdap-docs/tracker-manual/source/index.rst

```diff
@@ -133,8 +147,8 @@ To enable Tracker, go to the **Tracker UI** at
 (Distributed CDAP) http://host:dashboard-bind-port/ns/default/tracker/home 
 and press the ``"Enable Tracker"`` button to deploy and start Tracker.
```

Comment:
Is this still required? In 3.5.0, it was already running when I went to `http://localhost:9999/ns/default/tracker/home`